tests/test_discovery.py

```diff
@@ -45,13 +45,15 @@
 
 
 async def say_hello(other_actor):
-    await trio.sleep(0.4)  # wait for other actor to spawn
+    await trio.sleep(1)  # wait for other actor to spawn
```

Comment:
I had to bump this because otherwise `tractor.find_actor(other_actor)` would return `None`, unsure if this is an issue.

Reply:
Hmm. Maybe we should try using [`tractor.wait_for_actor()`](https://github.com/tgoodlet/tractor/blob/master/tractor/_actor.py#L891) here.

Note it's an *async context manager*.

Reply:
The test down below is the one using [wait_for_actor](https://github.com/tgoodlet/tractor/blob/9233a8ea053689487ec216ccc16ec77be84d2f8d/tests/test_discovery.py#L55), so both would be testing the same thing. Should this one be removed in favour of `say_hello_use_wait` only?